go-apps/meepctl/utils/kibana.go

```diff
@@ -112,9 +117,9 @@ func isKibanaReady(cobraCmd *cobra.Command) bool {
 }
 
 func uploadDefaultIndex(indexId string, cobraCmd *cobra.Command) error {
```

Comment:
Since the error returned by this function is not used, make it return true or false to indicate success or failure.
This will prevent creating error objects for nothing.

Reply:
fixed as part of my latest commit